### DISCUSSION TITLE
Has anyone actually been banned or terminated?

### DISCUSSION THREAD

**Discussion by yonigold1** [pyalarmdotcom/alarmdotcom#305] [Q&A]
Im curious to know if Alarm.com actually bans or locks people out of their accounts for polling updates too much. Ive done some reading of their terms and conditions it doesn't mention anything remotely related. I also did some googling and could find anything online about account banning or locking. If I could, I would set mine to refresh every one minute so my automations could work with home assistant.

**Comment by cfpandrade** [comment]
I've been pulling every 10seconds for the last year without problems

**Reply by marleyjaffe** [reply]
Thats great news. 900 seconds is crazy slow

**Comment by glen4cindy** [comment]
I know the last reply to this thread was a long time ago. I have been curious about this. It seems like I noticed what seemed like "a lot" of activity right before this stopped working.

I figured out how to fix it. There's a thread going that describes how to get into the docker container and modify a file.
I requested (from alarm.com) for them to deactivate 2FA for one user and that user is assigned to HA.

What about the 300 seconds for "websocket reconnect timeout?"

Is that okay?

Thanks in advance
